kubectl-rook-ceph.sh

```diff
@@ -179,6 +193,50 @@ function run_operator_restart_command () {
   kubectl --namespace "$NAMESPACE" rollout restart deploy/rook-ceph-operator
 }
 
+function path_cm_rook_ceph_operator_config() {
+  if [[ "$#" -ne 2 ]]; then
+    fail_error "require exactly 2 subcommand: $*"
+  fi
+  kubectl --namespace "$NAMESPACE" patch configmaps rook-ceph-operator-config --type json --patch  "[{ op: replace, path: /data/$1, value: $2 }]"
```

Comment:
Will this command work if there is not currently a value in the configmap? Some of the default settings are commented out.

Reply:
if the provided cli value is not present in configmap it will add that in configmap. And, I don't think we comment out some values in `configmaps rook-ceph-operator-config`.